Regular-Question-Answers/determine-the-winner-of-a-bowling-game.cpp

```diff
@@ -0,0 +1,59 @@
+// Author: Caner Aslan
+// Question Link : https://leetcode.com/problems/determine-the-winner-of-a-bowling-game/description/
+
+class Solution {
+public:
+    int isWinner(vector<int>& player1, vector<int>& player2) {
+        // get the number of turns 
+        int turns = player1.size();
+        
+        // initialize each player's score with the number of first pin they hit
+        int p1Score = player1[0]; 
+        int p2Score = player2[0];
+
+        for (int i = 1; i < turns; i++) {
+            // check for the second turn

```

Comment:
You make the control whether it is the second element or not in every iteration. This seemed a little bit inefficient to me. This part can be moved to outside of the for loop.